packages/create-webpack-app/src/index.ts

```diff
@@ -0,0 +1,80 @@
+// Cspell:ignore plopfile, plopfile.js
+import { Command } from "commander";
+import { resolve, dirname } from "path";
+import nodePlop from "node-plop";
+import { fileURLToPath } from "url";
+
+// eslint-disable-next-line node/no-missing-import
+import { onSuccessHandler, onFailureHandler, logger } from "./utils/logger.js";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const program = new Command();
+const plop = await nodePlop(resolve(__dirname, "./plopfile.js"));
+const defaultValues = {
+  init: {
+    projectPath: process.cwd(),
+    langType: "none",
+    devServer: true,
+    htmlWebpackPlugin: true,
+    workboxWebpackPlugin: true,
+    cssType: "CSS only",
+    isCSS: true,
+    isPostCSS: true,
+    extractPlugin: "No",
+    packageManager: "npm",
+  },
+};
+
+program
+  .version("1.0.0", "-v, --version")
+  .usage("[command] [options]")
+  .helpOption("-h, --help", "Display help for command")
+  .description("A CLI tool to generate a Webpack project");
+
+program
+  .command("init", { isDefault: true })
+  .aliases(["i", "n", "c", "create", "new"])
+  .description("Initialize a new Webpack project")
+  .argument("[projectPath]", "Path to create the project")
+  .option("-f, --force", "Skip the prompt and use the default values", false)
```

Comment:
we need to add `--template` option too. The default value would be `'default'`.

Reply:
If it's okay I would like to do this change along with the react template and generator in a seperate PR.